sources/Outputs/include/Job.h

```diff
@@ -138,13 +141,18 @@ class Job {
    */
   boost::shared_ptr<job::OutputsEntry> writeOutputs() const;
 
+  /**
+   * @brief tells if the chosenOutput variable is contained in the chosenOutputs array of the configuration json file
+   *
+   * @param chosenOutput the chosenOutput variable to check
+   *
+   * @returns true if chosenOutput is contained in the chosenOutputs array of the configuration file
+   */
+  bool isOutputChosen(std::string chosenOutput) const;
+
  private:
   static constexpr bool useStandardModels_ = true;  ///< use standard models in job entry and file
-#if _DEBUG_
```

Comment:
not related?

Reply:
all booleans related to outputs should be set in config object

Reply:
Il faut remettre les directives de préprocesseurs qui ont été supprimées ?